src/app/pipes/nodedisplay.ts

```diff
@@ -41,6 +41,21 @@ export class NodeTypePluralPipe implements PipeTransform {
 }
 
 
+@Pipe({name: 'nodeStoryUrl'})
+export class NodeStoryUrlPipe implements PipeTransform {
```

Comment:
You're not using this pipe as a pipe in any of the HTML templates, so it should just be a class function on `ArtifactsTableComponent`.